tf_agents/keras_layers/bias_layer.py

```diff
@@ -39,12 +37,14 @@ class BiasLayer(tf.keras.layers.Layer):
         shape `(batch_size, input_dim)`.
   """
 
-  def __init__(self, bias_initializer='zeros', **kwargs):
+  def __init__(self, bias_initializer='zeros', bias_regularizer=None, bias_constraint=None, **kwargs):
```

Comment:
Can you reformat so it doesn't go over 80 characters and each argument use one line. Ex:
def __init__(self, 
                     bias_initializer='zeros',
                     bias_regularizer=None,
                     bias_constraint=None, 
                     **kwargs):